src/S3/Features/S3Transfer/ProgressListenerHelper.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Aws\S3\Features\S3Transfer\Temp;
+
+class ProgressListenerHelper
```

Comment:
Maybe instead of helper we can calls this `ProgressListenerUtils`?

Reply:
This has been removed.